common/app/views/fragments/nav/editionPickerDropdown.scala.html

```diff
@@ -21,7 +21,7 @@
         id="edition-picker-toggle"
         aria-controls="edition-dropdown-menu"
         class="u-h dropdown-menu-fallback js-enhance-checkbox"
-        data-link-name="nav2 : topbar : edition-picker: toggle"
+        data-link-name="nav3 : topbar : edition-picker: toggle"
```

Comment:
See also these PRs for the `nav2` --> `nav3` change:
https://github.com/guardian/dotcom-rendering/pull/7893
https://github.com/guardian/frontend/pull/26213